docs/reference/server-configuration.md

```diff
@@ -368,7 +434,7 @@ key format set in this configuration if no explicit key format is declared in th
 
 ## `ksql.persistence.default.format.value`
 
-**Per query:** yes
```

Comment:
@rodesai I just noticed that we seem to list some ksql configs as being per-query already, which we don't currently have in the `queryLevelProperties` list but can be added to it -- ksql configs are not affected by consolidating queries into a single runtime and should continue to be settable via `SET`, where appropriate. However some of these make no sense to list as per-query and should be fixed to say `"No"`

Can you run down this list and sanity check my take on which of these should be added as query-level properties and which of these should be fixed?

Reply:
Note that we're presumably also missing these from the UI as well and will need to do a quick PR for that as well, once we have a finalized list of the query props. Filed https://confluentinc.atlassian.net/browse/KCI-1753